### PR TITLE
Print name of joint with wrong interface

### PR DESCRIPTION
**Pull request by bmagyar**
This can be super helpful when dealing with a broken urdf.

### CODE REVIEW

**Comment by bmagyar** [comment]
Seems all ok, not sure what makes the display of jenkins here red, while over there it's fine, there's a CMake warning that seems to have crept in from a previous PR (check the project builds)

**Comment by davetcoleman** [comment]
Build failed because of an unrelated warning we're trying to resolve in https://github.com/ros-simulation/gazebo_ros_pkgs/pull/537

File: gazebo_ros_control/src/default_robot_hw_sim.cpp [hunk 1]
```diff
@@ -177,7 +177,7 @@ bool DefaultRobotHWSim::initSim(
     else
     {
       ROS_FATAL_STREAM_NAMED("default_robot_hw_sim","No matching hardware interface found for '"
-        << hardware_interface );
+        << hardware_interface << " while loading interfaces for " << joint_names_[j] );
```
Comment (davetcoleman) [inline gazebo_ros_control/src/default_robot_hw_sim.cpp]: do you mind fixing the missing single quote ``'`` around hardware_interface while you are at it?

**Review by davetcoleman — APPROVED** [review]

**Comment by davetcoleman** [comment]
Build failed because of an unrelated warning we're trying to resolve in https://github.com/ros-simulation/gazebo_ros_pkgs/pull/537

**Comment by bmagyar** [comment]
Nice catch with the `'`, fixed!

**Pull request merged**